modules/billing-account/README.md

```diff
@@ -212,21 +213,63 @@ module "billing-account" {
 # tftest modules=1 resources=2 inventory=budget-monitoring-channel.yaml
 ```
 
+#### Budget factory
+
+This module also exposes a factory for billing budgets, that works in a similar way to factories in other modules: a specific folder is searched for YAML files, which contain one budget description per file. The file name is used to generate the key of the resulting map of budgets, which is merged with the one coming from the `budgets` variable. The YAML files support the same type of the `budgets` variable.
```

Comment:
very minor. Can we rephrase this? It's really difficult to understand imho

Reply:
try to read it slowly Luca :)